http-server/src/main/java/org/triplea/server/lobby/game/listing/GameListingWebsocket.java

```diff
@@ -25,11 +23,6 @@ private GameListingEventQueue getEventQueue(final Session session) {
         (GameListingEventQueue) session.getUserProperties().get(GAME_LISTING_EVENT_QUEUE));
   }
 
-  @OnClose
-  public void close(final Session session, final CloseReason closeReason) {
-    getEventQueue(session).removeListener(session.getId());
-  }
-
```

Comment:
Just so I get this correctly: Starting with this update we lazy-remove any closed sessions?

Reply:
Wouldn't this possibly result in a ghost entry for an extended amount of time until another change request is made?

Reply:
Bump

Reply:
Turns out the 'close' is called whether client or server closes. If we only do a lazy close, then it gets weird in test with just one connection (nothing closes), it also looked like chatter is less reliable to drop. Went with instead to use 'onClose' to close connections and no lazy close at all. IMO important to do the close in one place (DRY)